### DISCUSSION TITLE
"Bug Bounties" for selected issues

### DISCUSSION THREAD

**Discussion by pktiuk** [jellyfin/jellyfin#11428] [General] [answered]
Hello,   

I see that Jellyfin team is not asking for donations and is working only voluntarily, but I think it may be a good idea to allow users to place bounties on selected issued/bugs they are **really** interested in.

It would increase number of new and external contributions (like some students seeing this as a better alternative to side-job or freelancers). Moreover, it would be additional information for developers which features are demanded (there is a difference between "I would like this feature" and "I can pay for implementing this feature").

In terms of practical implementation of this idea, I think platforms like [Polar.sh](https://polar.sh/) could be used as a simple tool for managing bounties. Jellyfin team would also need to define clear rules for accepting contributions linked with bounties, but it seems to be worth the effort.

I am very curious what do you think about it. (As users and maintainers of this project).

PS
(I am not sure whether it is the best place for this kind of suggestion/discussion, but it seems to be the most accessible one)

PPS   
Similar discussions across the internet:    
[GitHub Community discussion](https://github.com/orgs/community/discussions/4517)  
https://github.com/ActivityWatch/activitywatch/issues/259 - discussion about funding other tools and ways of funding are mentioned
https://github.com/borgbackup/borg/issues/5900

**Comment by crobibero** [comment, marked as answer]
https://github.com/jellyfin/jellyfin/issues/1203#issuecomment-489736801

**Reply by pktiuk** [reply]
My bad, I was looking in discussions.

I will comment there.
